### DISCUSSION TITLE
Automatic return to home screen from submenu options

### DISCUSSION THREAD

**Discussion by LavanyaNiki** [microsoft/Dynamics365Commerce.Solutions#274] [Q&A]
We have requirement, when using transaction options that enter the submenu, upon completion of the desired action, it should be automatically return to the initial menu. 
 
Example: We have one operation called "Anular" which is a submenu

<img width="253" alt="image" src="https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/140384016/da69085a-fe11-4b28-b01a-74814cff6292">

 and clicking on this operation shows further operations. 

<img width="203" alt="image" src="https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/140384016/284ee1f4-def2-4d56-a226-da0a6992b08a">

If the user does the "Anular Transaction" operation, after completion, it should return to the initial menu "Anular" without hitting the return button.

We are looking for technical assistance to implement this requirement.

**Comment by zhangguanghuib** [comment]
I have provided a workaround code internal to Surya,  have you received that, does it work for you?

**Reply by LavanyaNiki** [reply]
@zhangguanghuib:  Thank you, workaround solution helped us in some extent. 
If we are in action tab workaround solution is working fine, but if we move to "Discounts" tab same solution not working.
